pkg/infoschema/builder.go

```diff
@@ -101,6 +101,8 @@ func (b *Builder) ApplyDiff(m meta.Reader, diff *model.SchemaDiff) ([]int64, err
 		return applyExchangeTablePartition(b, m, diff)
 	case model.ActionFlashbackCluster:
 		return []int64{-1}, nil
+	case model.ActionLogRestore:
+		return applyLogRestore(b, m, diff)
```

Comment:
this doesn't follow online schema change. it's possible some node doesn't load the diff in time, and breaks linearizability